app-modules/case/database/migrations/2023_03_06_000029_create_case_update_items_table.php

```diff
@@ -6,15 +6,18 @@
 
 class CreateCaseUpdateItemsTable extends Migration
 {
-    public function up()
+    public function up(): void
     {
-        Schema::create('case_update_items', function (Blueprint $table) {
+        Schema::create('case_updates', function (Blueprint $table) {
             $table->bigIncrements('id');
-            $table->string('update');
-            $table->string('internal');
+            $table->unsignedBigInteger('case_id')->nullable();
+            $table->text('update');
+            $table->boolean('internal');
             $table->string('direction');
             $table->timestamps();
             $table->softDeletes();
+
+            $table->foreign('case_id', 'case_fk_8136752')->references('id')->on('case_items');
```

Comment:
Do we need to specify the foreign key name here? Can we just do the following and let the index be generated for us?

```suggestion
            $table->foreignId('case_id')->references('id')->on('case_items');
```